app/src/testWebkit/java/org/mozilla/focus/webkit/matcher/DisconnectTest.java

```diff
@@ -53,7 +53,7 @@ public void cleanup() {
     // http://robolectric.org/getting-started/#note-for-linux-and-mac-users
     @Test
     public void matches() throws Exception {
-        final UrlMatcher matcher = UrlMatcher.loadMatcher(RuntimeEnvironment.application, R.raw.blocklist, new int[] { R.raw.google_mapping }, R.raw.entitylist);
+        final UrlMatcher matcher = UrlMatcher.loadMatcher(RuntimeEnvironment.application, R.raw.blocklist, new int[] { R.raw.google_mapping }, R.raw.entitylist, R.raw.abpindo_adservers);
```

Comment:
NIT: It's Disconnect and APDIndo test now. Also, maybe we can add some site to test this here too but I have no idea how we could start.

Reply:
I agree, maybe wrap three lists(block, white, entity) into one source. Then pass-in a list with two sources(Disconnect, ABPIndo). Probably do this in next patch.